app/views/govuk_component/related_items.raw.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if local_assigns[:sections] && !sections.blank? %>
   <aside class="govuk-related-items" data-module="track-click" role="complementary">
     <% sections.each_with_index do |section, section_index| %>
-      <h2
+      <h2 data-track-count="sidebarRelatedItemSection"
```

Comment:
@nickcolley I'd originally thought of doing something like this, but I was put off by exactly your observation: it could drift out of sync with what users see, which is exactly what we're trying to track here and therefore is of the utmost importance.

For example, in an early draft of this change, I overlooked the following line buried in the taxonomy sidebar: https://github.com/alphagov/static/blob/master/app/views/govuk_component/taxonomy_sidebar.raw.html.erb#L31

This sort of logic is already in our codebase, creating differences between `count` and what the user sees, and I think it's much safer to select for elements that we know we want to track, rather than assume a count from the content item.

Given that Vin is quite keen to start tracking this all as soon as possible, I'm going to go ahead and merge, but if you do disagree with this approach, please let me know and we can make any necessary changes.